Student/Student/Submissions/SubmissionRubric/RubricViewController.swift

```diff
@@ -172,7 +172,9 @@ class RubricViewController: UIViewController {
         ratingDesc.lineBreakMode = .byWordWrapping
         ratingStack.addArrangedSubview(ratingTitle)
         ratingStack.addArrangedSubview(ratingDesc)
-        if let index = model.selectedIndex {
+        if model.freeFormCriterionComments == true && model.hideRubricPoints == true {
+            container.isHidden = true
+        } else if let index = model.selectedIndex {
```

Comment:
Mind adding a test for this? Looks like a good time to add `RubricViewControllerTests`. Not requesting we add tests for the entire view controller, just this change.